packages/components/src/components/hds/super-select/index.hbs

```diff
@@ -0,0 +1,51 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{! NOTE: naming is temporary to not conflict with existing power-select wrapper }}
+<div class={{this.classNames}}>
+  {{! temp notes...
+    Required:
+    * onChange
+    * options or search
+  }}
+
+  {{#if (eq this.selectionType "single")}}
+    <PowerSelect
+      @ariaDescribedBy={{@ariaDescribedBy}}
+      @ariaInvalid={{@ariaInvalid}}
+      @ariaLabel={{@ariaLabel}}
+      @ariaLabelledBy={{@ariaLabelledBy}}
+      @disabled={{@disabled}}
+      @onChange={{@onChange}}
+      @options={{@options}}
+      @renderInPlace={{true}}
+      @searchEnabled={{@searchEnabled}}
+      @selected={{@selected}}
+      @verticalPosition={{@verticalPosition}}
+      ...attributes
```

Comment:
It seems to make more sense to spread attributes on these components vs. the wrapper div (Maybe I could also just get rid of the wrapper div too?)

Reply:
it would probably make sense to aply the attributes here and aim to remove the wrapper. in theory, we could develop a PowerSelect theme, instead of using overrides

Reply:
I tried removing the wrapper but couldn't get it to apply a custom class name to the underlying components directly so far.